pyta_server/models.py

```diff
@@ -1,12 +1,18 @@
 from pyta_server import db
 from sqlalchemy.sql import func
 
+class Submissions(db.Model):
+    identifier = db.Column(db.String(200), server_default="no-id", primary_key=True)
```

Comment:
Don't use the combination of two columns as a primary key, just stick with an `id` field (it's simpler).

Also, try to think of a more descriptive name than "identifier" for this column, what we have right now is too generic.